### DISCUSSION TITLE
归档 | 全自动解析 微博/微信 视频 | ZkeqのCoding日志

### DISCUSSION THREAD

**Discussion by zkeq** [zkeq/Coding#597] [Announcements]
https://icodeq.com/2022/03e4ec0968c8/ 

 Coding Blog

**Comment by Polexyqz** [comment]
音乐很好听，东西也很棒，很厉害！

**Comment by DERMTOP** [comment]
来学习了

**Comment by zkeq** [comment]
> @DERMTOP
> 来学习了

##### 🐟 这个评论系统的回复总是感觉不*直观，新建了另一个评论系统玩玩 

##### （啊打字特效好卡，我去关了.. 👀

**Comment by CCKNBC** [comment]
微博的偷走了
